index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Appl Old Version</title>
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/navbar.css">
+  </head>
+  <body>
+    <main id="container">
+      <header class="nav-bar">
+        <div id="nav-initial">
+          <a href="#">
+            <img id="apple-logo" src="https://upload.wikimedia.org/wikipedia/commons/9/9a/Logo_apple_pnh.png" alt="Apple Logo" />
+          </a>
+        </div>
+        <div><a href="#">Store</a></div>
+        <div><a href="#">Mac</a></div>
+        <div><a href="#">iPod</a></div>
+        <div><a href="#">iPhone</a></div>
+        <div><a href="#">iPad</a></div>
+        <div><a href="#">iTunes</a></div>
+        <div><a href="#">Support</a></div>
+        <div id="nav-final">
+          <img src="images/magnifier-white.png" alt="" id="search-icon" />
+          <input type="text" name="Search" value="" id="nav-search">
+        </div>
+      </header>
+      <section>
+        <div class="your-verse">
+          <img src="images/your_verse_title_small.png" alt="" />
+          <img src="images/your_verse_subhead_small.png" alt="" id="subhead" />
+          <div class="film-story">
+
+          </div>
+        </div>
+      </section>
+    </main>
+    <section>
+      <div class="wrapper">
+        <div class="inner-wrapper">
+          <a href="#"><img src="images/promo_iphone5s.jpg" alt="iPhone 5s" /></a>
+        </div>
+        <div class="inner-wrapper">
+          <a href="#"><img src="images/promo_iphone5c.jpg" alt="iPhone 5c" /></a>
+        </div>
+        <div class="inner-wrapper">
+          <a href="#"><img src="images/promo_30_years.jpg" alt="iPhone promo year" /></a>
+        </div>
+        <div class="inner-wrapper">
+          <a href="#"><img src="images/promo_supplier_responsibility.jpg" alt="iPhone Supplier" /></a>
```

Comment:
Good :+1: another suggested way of going about this section which could be also be `<aside>` would be too set the images as backgrounds of the container and wrap the `<div>` within `<a>` tags.